examples/config/base/go-uretprobe-counter/bytecode.yaml

```
@@ -0,0 +1,23 @@
+---
+apiVersion: bpfman.io/v1alpha1
+kind: UretprobeProgram
```

```suggestion
kind: UprobeProgram
```
It's still considered a UprobeProgram